app/models/reporting/housed.rb

```diff
@@ -179,13 +179,22 @@ class Housed < ReportingBase
       where(head_of_household: true)
     end
 
-    def self.available_subpopulations
-      {
-        youth: 'Youth (today)',
-        youth_at_search_start: 'Youth (at search start)',
-        youth_at_housed_date: 'Youth (at housed date)',
-        veteran: 'Veteran',
-      }.freeze
+    def self.available_subpopulations(report_klass = nil)
```

Comment:
Why not make this an instance method and override `self.available_subpopulations` in `ThReport` and `EsReport`?